### ISSUE TITLE
Badger storage | RunValueLogGC() is managed with a fatal error

### ISSUE THREAD

**Issue by pricelessrabbit** [ostafen/clover#145]
Hi all :) trying out the last version of the v2 branch and i got this error. The real issue is that inside clover storage, it's managed with a Fatal error. 

```go
  case <-ticker.C:
  err := store.db.RunValueLogGC(GCDiscardRatio)
  if err != nil && errors.Is(err, badger.ErrNoRewrite) {
	  log.Fatalf("RunValueLogGC(): %s\n", err.Error())
  }
  }
```

So as a result, my application exits every 5 mins with no way to recover from errors.
Is there a way to fix it? should i opening a PR with to allow for a non-fatal error log?

I also add a comment to the original issue about it.

**Comment by ostafen** [comment]
Yes, open a PR, please

**Comment by Shane-XB-Qian** [comment]
> Closed #145 as completed.

so this was solved?

-- 
shane.xb.qian

**Comment by ostafen** [comment]
@Shane-XB-Qian: yes
